src/main/kotlin/net/savagelabs/skyblockx/core/Island.kt

```diff
@@ -606,6 +635,16 @@ fun getIslandFromLocation(location: Location): Island? {
     return null
 }
 
+// This is the same as getIslandFromLocation but it returns an island if the "location" is "amount" of blocks outside of said island
+fun getIslandFromLocationWide(location: Location, amount: Int): Island? {
```

Comment:
Could be narrowed down to a one-liner with precisely the same speed.

Reply:
I know now, just, when I was making the method, I didn't. Thank you though.